### DISCUSSION TITLE
stories/Mice/

### DISCUSSION THREAD

**Discussion by giscus** [shrike-505/Nest#5] [Announcements]
# stories/Mice/

https://shrike505.cc/stories/Mice/

**Comment by Demetrius-Marlow** [comment]
梦开始的地方😭

**Comment by hua-deng** [comment]
写的好好，赞！

**Reply by shrike-505** [reply]
好好好

**Comment by Fufu142857** [comment]
#### 立意鲜明的寓言童话，感谢分享！
斗胆交流几个点：
+ 主人公虽然设定是一只只想“混”的平凡老鼠，但他的遣词造句像文豪转世，甚至知道“理想主义者”这样的人类词汇，我反而觉得它比族长更清醒，更不像鼠辈……
+ 族长的来历文中没有提及，我猜想是从实验室逃出来的小白鼠吗？
+ 尽管立意是完全不同的，但“老鼠学习人类文明改造鼠群”这一设定，带给我很强烈的熟悉感——舒克贝塔等老鼠系列（可能是我只熟悉这个，不知道这是不是历史上一个经典设定）里有非常多篇童话以此为立足点。这个设定真的很有意思。
+ **结尾是我最喜欢的地方**，这个对比太有力了，让人不寒而栗。

再次感谢给予的阅读体验！

**Reply by shrike-505** [reply]
可以可以😍很久没复盘过这一篇处女作了，凭记忆梳理下

- 毕竟是我写的，肯定更不像鼠辈（；可以看作是“从更高维度描绘的”普罗大众的形象浓缩到一个角色上了，也连同我想传达的，无论是各种细枝末节的思想，还是开头-大进攻前-结尾的思想反差，一并表现在这个角色身上了。
- 其实我最原本的设定，族长是个有着什么什么（没想好）悲惨过去的人类，被因为什么什么（没想好）惩罚放逐为一只老鼠了，深谙人类世界的复杂与只着眼现实，再由于自身的命运，看到了鼠族的潜力，决心推翻人类世界。但由于当时写的时候是应文学社征稿（），没有什么时间精力乃至想法去延拓这个设定，于是最后退而求其次将人设和主旨锚定在追求理想上了。
- 设定主要灵感来源自当时比较风靡互联网的鼠人称呼，也有你说的这方面的因素，郑渊洁没毛病，舒克贝塔九鼠复仇记都是很好的以老鼠为主人翁进行的故事，以小生物为主角的还有沙漠潜艇这种故事，当时都给我留下了很深的印象。
